src/layers/__tests__/GeoJson.spec.js

```diff
@@ -1,5 +1,5 @@
 import L from 'leaflet';
-import {GeoJson} from '../GeoJson';
+import { GeoJson } from '../GeoJson';
```

Comment:
Should we add tests for GeoJsonGroup as well?

Reply:
The tests are very limited already and I suggest that we rather spend time on the new maps gl engine.